src/components/doc/Sidebar.js

```diff
@@ -89,6 +89,11 @@ const DocSidebar = ({ location }) => {
               text="GitHub via Oauth"
               location={location}
             />
+            <SidebarItem
+              to="/docs/integrations/github-org/"
+              text="Loading Github Organization Teams"
```

Comment:
This label is quite a bit longer than the other items in the sidebar. Can you shorten it to "GitHub teams" and position it underneath the other GitHub related integrations?

![Screenshot 2021-03-19 at 14 56 35](https://user-images.githubusercontent.com/562403/111800064-8073b900-88c3-11eb-9b9c-2cf072ad8405.png)

Reply:
done